src/components/Navigation.js

```diff
@@ -4,13 +4,13 @@ import { NavLink } from 'react-router-dom'
 import { ReactComponent as Home } from '../icons/home.svg'
 import { ReactComponent as Filter } from '../icons/filter.svg'
 
-export default function Footer() {
+export default function Navigation() {
   return (
     <FooterStyled>
-      <NavStyled exact to="/" activeClassName="chosen">
+      <NavStyled exact to="/" activeClassName="chosen" data-cy="home">
```

Comment:
you can style your active navigation element inline with activeStyle="color", no more classes needed

Reply:
good tip, thank you!